rostime/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(rostime)
 
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
No particular reason, I just wanted to make sure the exact code in #48 compiled. Is there a non-C++11 equivalent to std::nextafter?